xhprof_lib/utils/xhprof_runs.php

```diff
@@ -107,22 +107,22 @@ public static function getDbClass()
   * 
   *
 CREATE TABLE `details` (
```

Comment:
Yeah, I will then create another pull request with this changes, but anyway , i replicated this, it's quite easy, 
Try it with your scheme where -  `server_id` char(3) NOT NULL default 't11',
And set $_xhprof['servername'] = 'myserver';
It will throw an error since the size it's different, from 3 to whatever .
Probably no one complains because they fixed it locally, or just use a 3 char or less server-id